src/components/tagApplyMenu/AposTagApply.vue

```diff
@@ -0,0 +1,240 @@
+      });
+      return {
+        creating: false,
+        searchValue: { data: '' },
+        searchStatus: {},
+        checkboxes,
+        myTags: this.tags,
+        searchInputValue: ''
+      }
+    },
+    computed: {
+      disabledCreate() {
+        const matches = this.myTags.filter((tag) => {
+          return tag.slug === this.searchInputValue;
+        });
+        if (matches.length) {
+          return true;
+        } else {
+          return false;
+        }
+      },
+      searchTags() {
+        if (this.creating) {
+          return;
+        }
+        if (this.searchInputValue.length > 2) {
+          return this.myTags.filter((tag) => {
+            return tag.slug.includes(this.searchInputValue);
+          });
+        } else {
+          return this.myTags;
+        }
+      },
+      createLabel() {
+        if (this.searchInputValue.length) {
+          return `Create tag "${this.searchInputValue}"`
+        } else {
+          return 'Create new tag'
+        }
+      },
+      searchField() {
+        return {
+          name: 'tagSearch',
+          label: 'Apply Tags',
+          placeholder: 'Tags...',
+          help: 'Find an existing tag or add a new one',
+          icon: (!this.searchTags || !this.searchTags.length) ? 'Pencil' : 'Magnify'
+        }
+      }
+    },
+    methods: {
+      create() {
+        if (!this.searchInputValue || !this.searchInputValue.length) {
+          this.creating = true;
+          this.searchValue.data = 'New Tag';
+          this.$refs.textInput.$el.querySelector('input').focus();
+          this.$refs.textInput.$el.querySelector('input').select();
+        } else {
+          this.$emit('createTag', this.searchInputValue);
+          const tag = {
+            label: this.searchInputValue,
+            slug: this.searchInputValue,
+            checked: this.applyTo
+          }
+          this.checkboxes[tag.slug] = createCheckbox(tag, this.applyTo)
+          this.myTags.unshift(tag);
+          this.creating = false;
+          this.emitState();
+        }
+      },
+      update(slug) {
+        const tag = this.myTags.find(tag => tag.slug === slug);
+        const box = this.checkboxes[slug];
+        if (!tag.checked) {
+          tag.checked = this.applyTo;
+          box.value.data.push(slug);
+        } else {
+          if (tag.checked.length === this.applyTo.length) {
+            // previously full check, unapply to all
+            delete tag.checked;
+            box.value.data = [];
+          } else {
+            // mixed check, check all
+            tag.checked = this.applyTo;
+            this.checkboxes[slug].value.data = [slug];
+            delete box.status.readOnly;
+            delete box.choice.indeterminate;
+          }
+        }
+
+        // done, emit
+        this.emitState();
+      },
+      emitState() {
+        this.$emit('update', this.myTags)
+      },
+      updateSearchInput(value) {
```

Comment:
Right now I can't type in exactly what I see in the list (title case) because it's expecting the slug version. If we're slugifying, we should normalize with the same method here.